Observer/Backend/AddCatalogToCart.php

```diff
@@ -0,0 +1,56 @@
+<?php
+ namespace Ziffity\Task\Observer\Backend;
+ use Magento\Catalog\Model\Product;
+ use Magento\Framework\Event\Observer;
+ use Magento\Framework\Event\ObserverInterface;
+
+
+ class AddCatalogToCart implements ObserverInterface
+ {
+    public function __construct(
+        \Magento\Framework\Message\ManagerInterface $message,
+        \Magento\Checkout\Model\Session $session,
+        \Magento\Framework\MessageQueue\PublisherInterface $publisher,
+        \Magento\Framework\Serialize\SerializerInterface $serializer,
+        \Ziffity\Task\Model\Custom $Cart,
+    
+       \Ziffity\Task\Model\Repository $Repository
+    )
+    
+        
+        {
+            $this->_messageManager = $message;
+            $this->_checkoutSession=$session;
+            $this->publisher = $publisher;
+            $this->serializer = $serializer;
+            $this->Cart = $Cart;
+            $this->Repository = $Repository;
```

Comment:
@NandhiniKarthi Bad naming convention for variables. Always start with lowercase or _